elasticsearch/examples/kubesphere/README.md

```diff
@@ -226,3 +159,84 @@ $ curl http://<your node ip>:30057/_cluster/health?pretty
   "active_shards_percent_as_number" : 100.0
 }
 ```
+
+### The reference configuration
+
+#### Master node
+
+```
+clusterName: "elasticsearch"
+nodeGroup: "master"
+
+roles:
+  master: "true"
+  ingest: "false"
+  data: "false"
+  ml: "false"
+  remote_cluster_client: "false"
+
+# tolerations for master node
+tolerations:
+- effect: NoSchedule
+  key: node-role.kubernetes.io/master
+  value: ''
+
+replicas: 3
+minimumMasterNodes: 2
+
+esJavaOpts: "-Xmx512m -Xms512m"
+
+resources:
+  requests:
+    cpu: "512m"
+    memory: "512Mi"
+  limits:
+    cpu: "1"
+    memory: "1Gi"
+
+```
+
+#### Data node
+
+```
+clusterName: "elasticsearch"
+nodeGroup: "data"
+
+roles:
+  master: "false"
+  ingest: "true"
+  data: "true"
+  ml: "false"
+  remote_cluster_client: "false"
+
+
+# tolerations for data node
+tolerations:
```

Comment:
No need to add toleration for master node.

ES cluster should not be scheduled to master node in the production scenario.

You can comment out the toleration part for debug purpose